docs/help/account/user-account-settings.md

```diff
@@ -0,0 +1,28 @@
+# User account settings
+
+Accessible from the Dashboard, the Account Settings page allows you to quickly modify certain elements related to your account.
+
+From this page, you can modify:
+
+- your first and last name
+- your password
+- your username
+
+## Modify Username
+
+Your username serves also as your slug, which is part of the URL of your documentation and hubs except if you use custom domains.
+Modify your username will impact your URLs
+
+## Delete your account
+
+If necessary, you can initiate the deletion of your Bump.sh account from this section. Deleting your account will result in the removal of all data related to you: documentation, hubs, etc...
```

Comment:
Should we add a word about the fact the telling us why they are deleting it is super useful to us?

Reply:
I have mixed feelings on this: I'm not sure it belongs in documentation even if it's a message we want to convey. We may have other ways to do that, which are less "intrusive".

Reply:
It's up to you 👍